.github/workflows/deployment.yml

```diff
@@ -94,6 +95,7 @@ jobs:
           REACT_APP_TENDERLY_PROJECT_NAME: ${{ secrets.REACT_APP_TENDERLY_PROJECT_NAME }}
           REACT_APP_MMI_BACKEND_BASE_URL: ${{ secrets.MMI_BACKEND_BASE_URL }}
           REACT_APP_MMI_ENVIRONMENT: ${{ secrets.MMI_ENVIRONMENT }}
+          REACT_APP_WALLETCONNECT_PROJECT_ID: ${{ secrets.REACT_APP_WALLETCONNECT_PROJECT_ID_STAGING }}
```

Comment:
Is staging the env here?

Reply:
yep, this is the deploy env var for the dev & stage
```
  deploy:
    name: Deploy Dev/Staging
```

in the line 143 we are using the production one:
```
          react-app-walletconnect-project-id: ${{ secrets.REACT_APP_WALLETCONNECT_PROJECT_ID }}
```